lib/mp4/transmuxer.js

```diff
@@ -828,7 +828,7 @@ CoalesceStream.prototype.flush = function(flushSource) {
     // video timeline for the segment
     for (i = 0; i < this.pendingMetadata.length; i++) {
       id3 = this.pendingMetadata[i];
-      id3.cueTime = clock.videoTsToSeconds(
+      id3.cueTime = clock.metadataTsToSeconds(
```

Comment:
This seems like a typo to me as we are passing all the correct parameters, just calling the wrong function, woops